Firebase/DynamicLinks/Public/FDLURLComponents.h

```diff
@@ -512,11 +512,14 @@ FIR_SWIFT_NAME(DynamicLinkComponents)
  * @param link Deep link to be stored in created Dynamic link. This link also called "payload" of
  *     the Dynamic link.
  * @param domain Domain of your App. This value must be equal to your assigned domain from Firebase
- *     Console.
+ *     Console. (e.g. xyz.page.link). Note that the domain scheme is required to be https and is
+ * assumed as such by this API.
```

Comment:
Just to confirm, this was always the case but we didn't document it as such, right?

Reply:
Correct.